s6/debian-root/etc/cont-init.d/20-start.sh

```diff
@@ -7,8 +7,11 @@ if [ "${PH_VERBOSE:-0}" -gt 0 ] ; then
     bashCmd='bash -e -x'
 fi
 
-$bashCmd /start.sh
-
+# Start dnsmasq for validate_env and gravity.sh
 dnsmasq -7 /etc/dnsmasq.d
+
+$bashCmd /start.sh
```

Comment:
nice! that should speed up startup and fix the "[✗] DNS resolution is currently unavailable" bit right?